dashboard/views.py

```diff
@@ -79,6 +100,16 @@ def file_access_within_week(request, course_id=0):
     week_num_end = int(request.GET.get('week_num_end','0'))
     grade = request.GET.get('grade','all')
 
+    # json for eventlog
+    data = {
+        "week_num_start": week_num_start,
+        "week_num_end": week_num_end,
+        "grade": grade,
+        "course_id": course_id
+    }
+    eventlog(request.user, EVENT_VIEW_FILE_ACCESS, None, json.dumps(data))
```

Comment:
where is this event is getting logged? in the DB or Inline( Doc says that can be done this way too)? if it is in DB then until the insert is done the Response is not sent to the view? Ideally this should be like a event log request be sent to the background thread and that process should be doing the insert with out blocking the original request.

Reply:
Events are stored in the database as eventlog_log table. It is also accessible from the admin interface.